hardware_interface/src/resource_manager.cpp

```diff
@@ -976,10 +976,13 @@ class ResourceStorage
             }
             data.limited = data.command;
             is_limited = limiters[joint_name]->enforce(data.actual, data.limited, desired_period);
-            RCLCPP_ERROR_THROTTLE(
-              get_logger(), *rm_clock_, 1000,
-              "Command '%s' for joint '%s' is out of limits. Command limited to %f - %d",
-              interface_name.c_str(), joint_name.c_str(), value, is_limited);
+            if (is_limited)
+            {
+              RCLCPP_ERROR_THROTTLE(
+                get_logger(), *rm_clock_, 1000,
+                "Command '%s' for joint '%s' is out of limits. Command limited to %f - %d",
+                interface_name.c_str(), joint_name.c_str(), value, is_limited);
```

Comment:
```suggestion
                "Command of at least one joint is out of limits (throttled log).");
```

Reply:
I would change it to _EXPRESSION instead of changing it  to nested

Reply:
Ok there isn't a combination of throttle and EXPRESSION.

This looks good to me